assets/src/edit-story/app/story/useStoryReducer/reducers/combineElements.js

```diff
@@ -81,15 +81,20 @@ function combineElements(state, { firstId, firstElement, secondId }) {
       }
     : {};
 
-  const mediaProps = objectPick(element, [
+  const propsFromFirst = [
     'type',
     'resource',
     'scale',
     'focalX',
     'focalY',
     'flip',
     'backgroundOverlay',
-  ]);
+  ];
+  // If the element we're dropping into is not background, maintain link, too.
+  if (secondElementPosition !== 0) {
```

Comment:
Could we do `!secondElement.isBackground` here instead?  That should work too, right?

Reply:
We could even do:

```js
  const propsFromFirst = [
    'type',
    'resource',
    'scale',
    'focalX',
    'focalY',
    'flip',
    'backgroundOverlay',
    // If the element we're dropping into is not background, maintain link, too.
    ...(!secondElement.isBackground && ['link']),
  ];
```

to simplify - not sure if it's cleaner though.